doc/usecases/core/UC_keyset_lookup_hierarchy.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Lookup hierarchy in `KeySet`
```

Comment:
I see a few problems here not solvable within one review iteration (which we both want). I think we should remove the use case for now and add it later when we discuss about the API changes. (Simply remove the file, and we revert it later.)

Reply:
What problems?

This describes `ksBelow` exactly. If you want, I'll add the bit about removing the keys, then it is `ksCut`. That is already public API, so there isn't really anything to discuss, if we want to document the use cases for the **current** API.